source/community/users-list.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Users list
+authors: didib
+page_classes: community
+no_container: true
+---
+
+# Users list
+
+The [users mailing list](https://lists.ovirt.org/archives/list/users@ovirt.org/) is the main place for getting help, asking questions, consulting about design/hardware/configuration/performance etc., and of course sharing your experience.
+
+## Getting help
+
+TL;DR: When asking for help, please provide: what you did, versions of packages used, content of `/var/log/*ovirt*` and `/var/log/vdsm/*` . Thanks!
+
```

Comment:
Also `yum -q list installed centos-release ovirt-release\* ovirt-engine redhat-release vdsm glusterfs` as per https://lists.ovirt.org/archives/list/users@ovirt.org/thread/GPCHNEZU56NVWBPFEOR6OF5KQF3T5NGS/

Reply:
Added this below. Above line is "TL; DR" - I don't want to make it any longer, perhaps better shorten if you can think of something...